package.json

```diff
@@ -1,15 +1,22 @@
 {
   "name": "salesforce-jwt-promise",
-  "version": "0.1.8",
+  "version": "0.2.0",
   "description": "Promise Based SalesForce Auth 2.0 JWT Bearer Token Flow Implementation, written in typescript",
   "main": "./bin/index.js",
   "typings": "bin/index.d.ts",
   "scripts": {
-      "prepublish": "tsc"
+    "prepublish": "./node_modules/.bin/tsc"
   },
   "dependencies": {
-    "jsonwebtoken": "^8.3.0",
-    "request": "^2.88.0"
+    "@types/node": "^13.7.0",
+    "@types/qs": "^6.9.0",
+    "axios": "^0.19.2",
+    "jsonwebtoken": "^8.5.1",
+    "node": "^13.7.0",
+    "qs": "^6.9.1",
+    "ts-custom-error": "^3.1.1",
+    "tslint": "^6.0.0",
+    "typescript": "^3.7.5"
```

Comment:
typescript, ts-lint and `@types` packages should be dev-dep.

Maybe also node?